symengine/lib/symengine_wrapper.pyx

```diff
@@ -791,6 +791,18 @@ cdef class Number(Basic):
 cdef class Integer(Number):
 
     @property
+    def is_positive(self):
+        return deref(symengine.rcp_static_cast_Integer(self.thisptr)).is_positive()
```

Comment:
But when I'm porting over to `SymPy` and an expression like 
```
order = Integer(3)
if order.is_positive:
```
comes up, an error is raised like this
```
AttributeError: 'symengine.lib.symengine_wrapper.Integer' object has no attribute 'is_positive'
```